api-docs/scoped-token.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Scoped Tokens
```

Comment:
Should the title just be API Tokens v2?
As an external developer I'm not sure I would understand what a scoped token is and why I would want it